releasenotes/notes/fix-tag-name-9f31ba1dce867c81.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Change added node tag name from node_role to kube_node_role.
```

Comment:
Not sure I properly understand this changelog 😅

Reply:
Uh, yeah that does sound bad 😓 . Let me rephrase it. 
`Rename node role tag from "node_role" to "kube_node_role"`